task-ballerina/Dependencies.toml

```diff
@@ -0,0 +1,4 @@
+[[dependency]]
+org = "ballerina"
+name = "time"
+version = "1.0.8"
```

Comment:
Don't use hardcoded versions in the `Dependencies.toml` or `Ballerina.toml` files. Check how other packages use placeholders and then replace them at the build time.